### PR TITLE
LOVE-403 - Improvements to e-commerce-microservice blueprint

### PR DESCRIPTION
**Pull request by jkrieger-xl**

### CODE REVIEW

File: aws/microservice-ecommerce/blueprint.yaml [hunk 1]
```diff
@@ -63,6 +63,10 @@ parameters:
   type: Input
   description: Store Admin password
   value: admin
+- name: XLDUrlForXLR
```
Comment (deepu105) [inline aws/microservice-ecommerce/blueprint.yaml]: why is this needed? we were actually getting rid of such stuff

Comment (jkrieger-xl) [reply aws/microservice-ecommerce/blueprint.yaml]: XLD and XLR are inside the docker cluster, there was an XLR step failing because it was trying to access XLD using 'localhost'
